pkg/apis/resources/v1alpha1/types.go

```diff
@@ -145,6 +145,16 @@ const (
 	// PodTopologySpreadConstraintsSkip is a constant for a label on a Pod which indicates that this Pod should not be considered for
 	// adding the pod-template-hash selector to the topology spread constraint.
 	PodTopologySpreadConstraintsSkip = "topology-spread-constraints.resources.gardener.cloud/skip"
+
+	// NetworkingCrossNamespaceSelectors is a constant for an annotation on a Service which contains a list of namespace
+	// selectors.
+	NetworkingCrossNamespaceSelectors = "networking.resources.gardener.cloud/cross-namespace-selectors"
```

Comment:
What about just "namespace-selectors"? This somehow fits to what happens with `NetworkPolicy`s, i.e., by default, only the same namespace is  respected - unless you explicitly specify a namespace selector for ingress/egress.
With this controller it would be the same then.

Reply:
Indeed, `namespace-selectors` sounds better. Works for me.